### PR TITLE
Add support for random collection element selection

### PR DESCRIPTION
**Pull request by denniss17**
This PR adds methods to select random elements from arrays, collections or enumerations.

Fixes https://github.com/DiUS/java-faker/issues/211

### CODE REVIEW

File: src/main/java/com/github/javafaker/service/RandomService.java [hunk 1]
```diff
@@ -50,4 +51,40 @@ public double nextDouble() {
...
     }
+
+    /**
+     * Returns a random element from an array.
+     *
+     * @param array The array to take a random element fom.
+     * @param <E>   The type of the elements in the array.
+     * @return A randomly selected element from the array.
+     */
+    public <E> E nextElement(E[] array)
```
Comment (codingricky) [inline src/main/java/com/github/javafaker/service/RandomService.java]: sorry to be a pain but you could fix up the formatting, thanks

Comment (denniss17) [inline src/main/java/com/github/javafaker/service/RandomService.java]: Yep sorry, normally I'm using another code format, my bad.

File: src/main/java/com/github/javafaker/service/RandomService.java [hunk 2]
```diff
@@ -50,4 +51,37 @@ public double nextDouble() {
...
+    }
+
+    /**
+     * Returns a random enumeration value
+     *
+     * @param enumeration The enumeration to take a random value from
+     * @param <E>         The type of the enumeration
+     * @return A randomly selected enumeration value
+     */
+    public <E extends Enum<E>> E nextEnumValue(Class<E> enumeration) {
```
Comment (PascalSchumacher) [inline src/main/java/com/github/javafaker/service/RandomService.java]: just wanted to mention, that this duplicates https://github.com/DiUS/java-faker/blob/master/src/main/java/com/github/javafaker/Options.java#L15

Comment (denniss17) [inline src/main/java/com/github/javafaker/service/RandomService.java]: Hmm you're right. However, I would expect this method to be under `random()`, not under `option()`. One of these is indeed obsolete, the question is which one ;)

Comment (codingricky) [inline src/main/java/com/github/javafaker/service/RandomService.java]: @denniss17 sorry to waste your time, we'll just keep the existing implementation as is. Thanks @PascalSchumacher

**Comment by PascalSchumacher** [comment]
@codingricky But what about the other methods added by @denniss17?

**Comment by codingricky** [comment]
@PascalSchumacher @denniss17 moved the new methods into the options class.

**Comment by denniss17** [comment]
But you still merged this PR? So it is not moved to the options class?

**Comment by denniss17** [comment]
Ah I see, you did a follow up

**Pull request merged**